<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.context.support.PropertyResourceConfigurerIntegrationTests" tests="7" skipped="0" failures="0" errors="0" timestamp="2021-03-04T16:39:50" hostname="WIN-LQAMLJ4MN5U" time="0.03">
  <properties/>
  <testcase name="testPropertyPlaceholderConfigurerWithSystemPropertyInLocation()" classname="org.springframework.context.support.PropertyResourceConfigurerIntegrationTests" time="0.008"/>
  <testcase name="testPropertyPlaceholderConfigurerWithMultiLevelCircularReference()" classname="org.springframework.context.support.PropertyResourceConfigurerIntegrationTests" time="0.002"/>
  <testcase name="testPropertyPlaceholderConfigurerWithSystemPropertiesInLocation()" classname="org.springframework.context.support.PropertyResourceConfigurerIntegrationTests" time="0.007"/>
  <testcase name="testPropertyPlaceholderConfigurerWithNestedCircularReference()" classname="org.springframework.context.support.PropertyResourceConfigurerIntegrationTests" time="0.002"/>
  <testcase name="testPropertyPlaceholderConfigurerWithNestedUnresolvableReference()" classname="org.springframework.context.support.PropertyResourceConfigurerIntegrationTests" time="0.002"/>
  <testcase name="testPropertyPlaceholderConfigurerWithUnresolvableSystemPropertiesInLocation()" classname="org.springframework.context.support.PropertyResourceConfigurerIntegrationTests" time="0.006"/>
  <testcase name="testPropertyPlaceholderConfigurerWithValueFromSystemProperty()" classname="org.springframework.context.support.PropertyResourceConfigurerIntegrationTests" time="0.002"/>
  <system-out><![CDATA[00:39:50.733 [Test worker] WARN  o.s.c.s.StaticApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanInitializationException: Could not load properties; nested exception is java.io.FileNotFoundException: class path resource [D:/tools/source/spring-framework-5.2.x/spring-context/test] cannot be opened because it does not exist
00:39:50.735 [Test worker] WARN  o.s.c.s.StaticApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanDefinitionStoreException: Invalid bean definition with name 'tb1' defined in null: Circular placeholder reference 'var' in property definitions; nested exception is java.lang.IllegalArgumentException: Circular placeholder reference 'var' in property definitions
00:39:50.742 [Test worker] WARN  o.s.c.s.StaticApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanInitializationException: Could not load properties; nested exception is java.io.FileNotFoundException: class path resource [D:/tools/source/spring-framework-5.2.x/spring-context/test/D:/tools/source/spring-framework-5.2.x/spring-context] cannot be opened because it does not exist
00:39:50.745 [Test worker] WARN  o.s.c.s.StaticApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanDefinitionStoreException: Invalid bean definition with name 'tb1' defined in null: Circular placeholder reference 'm' in property definitions; nested exception is java.lang.IllegalArgumentException: Circular placeholder reference 'm' in property definitions
00:39:50.747 [Test worker] WARN  o.s.c.s.StaticApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanDefinitionStoreException: Invalid bean definition with name 'tb1' defined in null: Could not resolve placeholder 'm2' in value "${m2}"; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'm2' in value "${m2}"
00:39:50.752 [Test worker] WARN  o.s.c.s.StaticApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanInitializationException: Could not load properties; nested exception is java.io.FileNotFoundException: class path resource [${myprop}/test/${myprop}] cannot be opened because it does not exist
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
